src/Controller/EnMarche/SummaryManagerController.php

```diff
@@ -37,14 +39,39 @@ public function indexAction()
         $this->disableInProduction();
 
         $member = $this->getUser();
+        $summary = $this->get(SummaryManager::class)->getForAdherent($this->getUser());
+        $pathImage = 'images/'.$summary->getMemberUuid().'.jpg';
```

Comment:
Maybe we could add a boolean flag on the entity to know that the CV has a picture, it would allow to count it for completion (it's currently ignored at 100%), and would avoid this logic if not necessary.

ping @tgalopin @marineb